src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -143,15 +142,18 @@ class NotebookPreviewFrame extends Component {
   }
 
   render() {
-    const { notebookName, workspace, queryParams }  = this.props
+    const { loadedChildren } = this.props
     const { preview, busy } = this.state
     return h(Fragment, [
-      !!queryParams['read-only'] && h(ReadOnlyMessage, { notebookName, workspace }),
+      preview && loadedChildren,
       preview && iframe({
         style: { border: 'none', flex: 1 },
         srcDoc: preview
       }),
-      busy && spinnerOverlay
+      busy && div({ style: { ...styles.step.container, marginLeft: '2rem' } }, [
```

Comment:
Since this spinner can last for a little while, I thought I'd add some explanation